### PR TITLE
kubevirt, sig-buildsystem: add charter and entry in sigs.yaml

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

Adds the `sig-buildsystem` definition and adds the sig to the sigs.yaml

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/cc @brianmcarey @xpivarc

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
add sig-buildsystem
```

### CODE REVIEW

File: sig-buildsystem/charter.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This also includes the support of failing builds or test lanes whose failures are caused by infrastructural or configuration problems.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- all test automation scripts inside folder `automation/`
+- KubeVirt builder and other build related scripts inside `hack/`
+- `cluster-up/`, [KubeVirtCI] updates
+
```
Comment (xpivarc) [inline sig-buildsystem/charter.md:16]: What about other repositories?

**Comment by dhiller** [comment]
@xpivarc @fabiand thank you for your remarks. I've extended the scope to cover with more concrete things, hope it's better now. PTAL!

File: sig-buildsystem/charter.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# SIG BUILDSYSTEM Charter
```
Comment (fabiand) [inline sig-buildsystem/charter.md:1]: I think "Buildsystem" (no capital case) is ok here.

**Review by fabiand — COMMENTED** [review]
lgtm! Thanks

**Comment by fabiand** [comment]
@dhiller  how can we converge aka merge this?

**Comment by dhiller** [comment]
> @dhiller how can we converge aka merge this?

From my POV there are no open issues on this any more. Or are there? @brianmcarey do you see anything that you would like to be changed?

**Review by brianmcarey — COMMENTED** [review]
Looks good to me - thanks @dhiller 

/lgtm

**Pull request merged**